card/lib/card/query.rb

```diff
@@ -61,7 +61,8 @@ class Query
                            link_to linked_to_by
                            include included_by                                ),
       conjunction:     %w(and or all any),
-      special:         %w(found_by not sort match complete extension_type),
+      special:         %w(found_by not sort match name_match complete junction_complete
```

Comment:
yeah, that would have been a lot easier to find if it were in its own method, eh?